Tests/RxFeedbackTests/RxFeedbackObservableTests.swift

```diff
@@ -330,6 +330,43 @@ extension RxFeedbackObservableTests {
             "initial_a_b_c"
             ])
     }
+
+    func testUIBindingsAreNotDisposedWhenNoEventsAreSpecified() {
```

Comment:
über test naming :P

Reply:
I'm used to reading `Quick` test names, so take a no holds barred approach :)

Reply:
Not sure how to go about the circle CI error